odxtools/write_pdx_file.py

```diff
@@ -11,61 +11,85 @@
 
 from typing import Any, Dict, List, Tuple
 
-def jinja2_odxraise_helper(msg):
+from .exceptions import OdxError
+from .comparam_subset import BaseComparam, Comparam, ComplexComparam
+
+odxdatabase = None
+
+def jinja2_odxraise_helper(msg: str) -> None:
     raise Exception(msg)
 
+def comparam_subset_name(x: BaseComparam) -> str:
+    """Find the short name of the comparam subset which contains a
+    given communication parameter"""
```

Comment:
why not just use the doc frag of the odx_id?

Reply:
good point